packages/sync/src/Actions.php

```diff
@@ -263,24 +385,47 @@ static function do_full_sync( $modules = null ) {
 		return true;
 	}
 
-	static function jetpack_cron_schedule( $schedules ) {
+	/**
+	 * Adds a cron schedule for regular syncing via cron, unless the schedule already exists.
+	 *
+	 * @access public
+	 * @static
+	 *
+	 * @param array $schedules  The list of WordPress cron schedules prior to this filter.
+	 * @return array            A list of WordPress cron schedules with the Jetpack sync interval added.
+	 */
+	public static function jetpack_cron_schedule( $schedules ) {
 		if ( ! isset( $schedules[ self::DEFAULT_SYNC_CRON_INTERVAL_NAME ] ) ) {
+			$minutes = intval( self::DEFAULT_SYNC_CRON_INTERVAL_VALUE / 60 );
+			$display = ( 1 === $minutes ) ?
+				__( 'Every minute', 'jetpack' ) :
+				/* translators: %d is an integer indicating the number of minutes. */
+				sprintf( _n( 'Every %d minute', 'Every %d minutes', $minutes, 'jetpack' ), $minutes );
```

Comment:
`Every %d minute` will never be used that way. Should we remove it, altogether with the `_n()` call and leave it just a `__()`?